### DISCUSSION TITLE
notes/%E9%AB%98%E5%B9%B6%E5%8F%91%E7%B3%BB%E7%BB%9F%E5%AE%9E%E6%88%98/%E5%AF%B9%E6%AF%94%E5%8D%95%E4%BD%93%E7%B3%BB%E7%BB%9F%E3%80%81%E5%88%86%E5%B8%83%E5%BC%8F%E7%B3%BB%E7%BB%9F%E5%92%8C%E5%BE%AE%E6%9C%8D%E5%8A%A1%E7%B3%BB%E7%BB%9F

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#507] [Announcements]
# notes/%E9%AB%98%E5%B9%B6%E5%8F%91%E7%B3%BB%E7%BB%9F%E5%AE%9E%E6%88%98/%E5%AF%B9%E6%AF%94%E5%8D%95%E4%BD%93%E7%B3%BB%E7%BB%9F%E3%80%81%E5%88%86%E5%B8%83%E5%BC%8F%E7%B3%BB%E7%BB%9F%E5%92%8C%E5%BE%AE%E6%9C%8D%E5%8A%A1%E7%B3%BB%E7%BB%9F

单体系统之痛 单体系统并非一无是处。在某些场景中，单体系统是最佳的选择，可以帮助企业业务快速发展。 在企业初创期，投入较少的研发资源构建出；适应当前业务的单体应用，从而达到抢占市场和技术试错的目的。 什么是单体系统 单体系统即一个应用程序，所有的业务代码都在这一个应用程序中，所有的表也都在一个数据库中，所涉及的相关文件都在同一个服务器上。 在企业初创期...

https://stiflea.top/notes/%E9%AB%98%E5%B9%B6%E5%8F%91%E7%B3%BB%E7%BB%9F%E5%AE%9E%E6%88%98/%E5%AF%B9%E6%AF%94%E5%8D%95%E4%BD%93%E7%B3%BB%E7%BB%9F%E3%80%81%E5%88%86%E5%B8%83%E5%BC%8F%E7%B3%BB%E7%BB%9F%E5%92%8C%E5%BE%AE%E6%9C%8D%E5%8A%A1%E7%B3%BB%E7%BB%9F.html

**Comment by stiflea** [comment]
hello

**Reply by Mister-Hope** [reply]
# Since I already warned you, you are blocked
